app/service/data_svc.py

```diff
@@ -79,21 +79,20 @@ def __init__(self):
 
     async def load_data(self):
```

Comment:
There's nothing in this function that requires the use of the `async` keyword. 

Also, this presents another race condition in the server startup. Line 51 in server.py `loop.run_until_complete(data_svc.load_data())`. 

The event loop will run until the Future that is returned is completed, but because no Future is returned, it's assumed to be complete, and you are not guaranteed that all the data is loaded at that point.